tests/Fixer/ControlStructure/SwitchContinueToBreakFixerTest.php

```diff
@@ -33,6 +33,9 @@ public function testFix(string $expected, ?string $input = null): void
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @return iterable<array{0: string, 1?: string}>
```

Comment:
can we use following ?

```suggestion
     * @return iterable<string, array{0: string, 1?: string}>
```

Reply:
No:
```console
 ------ -----------------------------------------------------------------
  Line   tests/Fixer/ControlStructure/SwitchContinueToBreakFixerTest.php
 ------ -----------------------------------------------------------------
  269    Generator expects key type string, int given.
         🪪  generator.keyType
  432    Generator expects key type string, int given.
         🪪  generator.keyType
 ------ -----------------------------------------------------------------
```

Reply:
aaaaand, especially one of them is from code you touch in this PR

I see no reason why not to fix that single case that was missing the description.

#8546